api-catalog-ui/frontend/src/setupTests.js

```diff
@@ -1,13 +1,4 @@
-/*
```

Comment:
Any specific reason to remove the license?

Reply:
Removed while debugging babel compilation errors, forgot to add back. Resolved.